packages/@aws-cdk/aws-ecs-patterns/test/fargate/test.load-balanced-fargate-service.ts

```diff
@@ -29,6 +29,35 @@ export = {
     test.done();
   },
 
+  'setting public-facing NLB with fargate with assignPublicIp creates securigy group allow all'(test: Test) {
+    // GIVEN
+    const stack = new cdk.Stack();
+    const vpc = new ec2.Vpc(stack, 'VPC');
+    const cluster = new ecs.Cluster(stack, 'Cluster', { vpc });
+
+    // WHEN
+    new ecsPatterns.NetworkLoadBalancedFargateService(stack, 'Service', {
+      cluster,
+      publicLoadBalancer: true,
+      assignPublicIp: true,
+      taskImageOptions: {
+        image: ecs.ContainerImage.fromRegistry("amazon/amazon-ecs-sample"),
+      },
+    });
+
+    // THEN
+    expect(stack).to(haveResourceLike('AWS::EC2::SecurityGroup', {
```

Comment:
Apologies if I'm missing something, but would we not want to assert on the SecurityGroupIngress rather than Egress?